### DISCUSSION TITLE
Water Meter

### DISCUSSION THREAD

**Discussion by gideonklein** [indomus/forum#2537] [GENERALE - Discussioni generiche]
`Buongiorno a tutti,

altro problema con sensore acqua ed esphome.
Il contatore acqua è questo: https://www.amazon.it/dp/B082HQ3WL5?psc=1&ref=ppx_yo2ov_dt_b_product_details
La board è esp8266: d1_mini
Ecco il codice con i sensori:
```
sensor:
  - platform: pulse_counter
    id: water_pulse
    pin: D2
    internal: False
    name: "Contatore di impulsi x"
    update_interval: 1s
    filters:
    - lambda: |-
        static float total_pulse = 0.0;
        total_pulse += x * 1 / 60.0;
        id(lifetime_counter).publish_state(total_pulse / 225);
        return x ;

  - platform: template
    id: lifetime_counter
    name: "Acqua x"
    unit_of_measurement: "L"
    device_class: water
    state_class: total_increasing    
    icon: "mdi:hand-water"
    accuracy_decimals: 2

  - platform: template
    name: "Flusso acqua x"
    id: water_flow_rate
    accuracy_decimals: 1
    unit_of_measurement: "L/min"
    icon: "mdi:water"
    lambda: return (id(water_pulse).state / 225);
    update_interval: 1s
```

Da circa 1 mese non funziona piu'. Praticamente legge in continuazione il flusso d'acqua come fosse sempre aperta. Sicuramente dopo qualche aggiornamento è cambiato qualcosa ma non capisco cosa. Il contatore funziona perfettamente e anche il D1 mini (ne ho provati altri). Quindi è la configurazione. Qualcuno può aiutarmi?

**Comment by sky7176** [comment]
Ciao hai provato a vedere con un tester l'uscita del sensore?
Giusto per capire se non è saltato il sensore.

Oppure fatti inviare, non conosco esphome, solo il conteggio del sensore, giusto per capire cosa ti arriva
